build.gradle

```
@@ -1,6 +1,6 @@
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 plugins {
-    id("org.jetbrains.intellij.platform") version "2.3.0"
+    id("org.jetbrains.intellij.platform") version "2.5.0"
```

```suggestion
    id("org.jetbrains.intellij.platform") version "2.5.0"
```

In the meantime there was again an update of the version